### PR TITLE
Updated the ./assets/logos to match split.io rebrand done in May 2023.

### PR DESCRIPTION
**Pull request by dtk-DaveKarow**
### What does this PR do?

Just updating the logos to match split.io's latest brand.

### Motivation

Fulfilling request from Meghan Lo.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Review by yzhan289 — APPROVED** [review]

**Comment by drichards-87** [comment]
Created DOCS-5862 for Docs Team review.

**Comment by dtk-DaveKarow** [comment]
Follow-up:
1. Realized you might want the assets folder cloned for the split-rum tile, as each tile probably needs to be self-sufficient and not rely on other tiles for assets, right?  If so, the contents are in split/assets. 
2. Apologize as it seems I've appended to PR 1889 when my intention was to open a separate PR for creating the split-rum tile that goes alongside the existing split tile.  Please advise if you can work these together or whether I need to start over for split-rum.

File: split-rum/CHANGELOG.md
Comment (anawishnoff) [inline split-rum/CHANGELOG.md:1]:
```suggestion
# CHANGELOG - Split
```

File: split-rum/README.md
Comment (anawishnoff) [inline split-rum/README.md:1]:
```suggestion
```
This can be removed since it is not an agent check

File: split-rum/README.md
Comment (anawishnoff) [inline split-rum/README.md:15]:
```suggestion
3. Initialize Split’s SDK and create an impression listener, reporting feature flag evaluations to Datadog using the following snippet of code:

```

File: split-rum/images/placeholder.txt
Comment (anawishnoff) [inline split-rum/images/placeholder.txt:1]: As stated, this can be deleted

File: split-rum/manifest.json
Comment (anawishnoff) [inline split-rum/manifest.json:15]: I noticed a slight typo in this image - inferred has an extra 'r' :)

File: split-rum/manifest.json
Comment (anawishnoff) [inline split-rum/manifest.json:3]: You can use a uuid generator for this: https://www.uuidgenerator.net/

File: split-rum/manifest.json
Comment (anawishnoff) [inline split-rum/manifest.json:12]:
```suggestion
    "title": "Split - RUM",
```

**Review by anawishnoff — CHANGES_REQUESTED** [review]
This looks good so far, a few comments from my end!

File: split-rum/manifest.json
Comment (dtk-DaveKarow) [reply split-rum/manifest.json:15]: Good catch.  Thank you!
I fixed this in latest version of split-rum/images/split-feature-flags-list-in-dd-rum.png committed a moment ago.

**Review by anawishnoff — APPROVED** [review]

File: split-rum/manifest.json
Comment (james-eichelbaum) [inline split-rum/manifest.json:6]: Could you add some uninstallation instructions in the readme and add a section here linking to them?

**Review by james-eichelbaum — CHANGES_REQUESTED** [review]

**Comment by dtk-DaveKarow** [comment]
@anawishnoff looks like as soon as @james-eichelbaum marks his requested change accepted, we are clear to merge.  Is that correct?  I see that he edited his request to strike it out, but Git seems to want him to mark it accepted/resolved anyway...

**Comment by anawishnoff** [comment]
Hi @dtk-DaveKarow , we are also still waiting on reviews from our Documentation and Agent Integrations team. I've pinged the teams internally to submit a review.

**Review by james-eichelbaum — CHANGES_REQUESTED** [review]
Still one more comment around the support section if you wouldn't mind updating that one!

File: split-rum/README.md
Comment (buraizu) [inline split-rum/README.md:10]:
```suggestion
Feature flag tracking is available in the RUM Browser SDK. For detailed setup instructions, see the [Getting started with Feature Flag data in RUM][3] guide.
```

File: split-rum/README.md
Comment (buraizu) [inline split-rum/README.md:12]:
```suggestion
1. Update your Browser RUM SDK version to 4.25.0 or later.
```

File: split-rum/README.md
Comment (buraizu) [inline split-rum/README.md:14]:
```suggestion
3. Initialize the Split SDK and create an impression listener to report feature flag evaluations to Datadog using the following snippet of code:
```

File: split-rum/README.md
Comment (buraizu) [inline split-rum/README.md:39]:
```suggestion
Need help? See the [JavaScript SDK page][4] in the Split documentation or contact [Datadog Support][5].
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, just left some minor suggestions per the documentation [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md). Please let me know if you have any questions.

**Comment by dtk-DaveKarow** [comment]
@anawishnoff got and accepted great edits from @buraizu and @james-eichelbaum.  Built tests all pass.  I *think* we just need re-review from @buraizu and @james-eichelbaum, which I just requested.

**Review by buraizu — APPROVED** [review]
Thanks, looks good from a docs perspective!

**Review by james-eichelbaum — APPROVED** [review]

**Pull request merged**
